decidim/decidim
# Add new statistics design to Assemblies

#### :tophat: What? Why?

When reviewing the application I found out that we haven't implemented the new design for statistics in some spaces: 

* Participatory Processes have the '.process-stats' cell kind of stats
* Assemblies and Conferences have the '.process_stats' partial kind of stats 
* There were also some missing strings on some components (see Screenshot below)


This PR: 

* Renames the '.process-stats' to '.space-stats' so the naming is more generic
* Implements this is Assemblies 
* Adds the strings

After this is reviewed and merged, then the next steps will be:
* to implement this in Conferences
* to move the logic to `decidim-core` and DRY this up

#### :pushpin: Related Issues

- Related to #5688


#### Testing

1. Log in as admin
2. Go to an assembly, enable "Show statistics"
3. Go to the home of this Assembly. See the statistics design.

#### :clipboard: Checklist
:rotating_light: Please review the [guidelines for contributing](../CONTRIBUTING.adoc) to this repository.


### :camera: Screenshots

#### Before
![imatge](https://user-images.githubusercontent.com/717367/106712713-a128cd80-65f9-11eb-9dec-94b489b0cf3f.png)

#### After

![imatge](https://user-images.githubusercontent.com/717367/106712723-a4bc5480-65f9-11eb-91b5-3f52fbdb504f.png)


:hearts: Thank you!
